src/App.js

```diff
@@ -1,12 +1,38 @@
+import {
+  BrowserRouter, Routes, Route, Outlet, Link,
+} from 'react-router-dom';
+import './components/Quotes.css';
 import './App.css';
+import { Home } from './components/Home';
+import Quotes from './components/Quotes';
 import Calculator from './components/Calculator';
 
+function Layout() {
+  return (
+    <>
+      <nav>
+        <ul className="navbar">
+          <li><Link to="/">Home</Link></li>
+          <li><Link to="/quotes">Quotes</Link></li>
+          <li><Link to="/calculator">Calculator</Link></li>
```

Comment:
**You did great job!, I realy like the implementation you did.**

- [ ] However, when it comes to navigation it looks like doesn't  match to the given wireframe. Please, change the order Quotes with Calculator :)